ansible/group_vars/all/defaults.yaml

```diff
@@ -5,11 +5,11 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.30.5"
+kubernetes_version: "1.31.3"
```

Comment:
Line 24 `crictl_sha256` will need to change too.

Reply:
done